Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -169,15 +175,18 @@ extension SettingsViewController: UITableViewDelegate {
   }
 
   func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
-    guard section == SettingsSectionType.ratingAppVersion.rawValue else { return nil }
+    guard let section = SettingsSectionType(rawValue: section), section.hasSectionFooter else { return nil }
```

Comment:
Since the logic is identical to the `heightForFooterInSection` method, we could create a private helper function  that returns a Bool and use the return value to return the height / view. 
```
private func shouldHideFooterHeaderView(for section: Int) -> Bool {
guard let section = SettingsSectionType(rawValue: section), section.hasSectionFooter else { return true }
    if section == SettingsSectionType.findFriends
      && !self.viewModel.outputs.findFriendsDisabledProperty.value {
      return true
    }
return false
}
```
and do `return shouldHideFooterHeaderView(for: section) ? 0.1 : UITableView.automaticDimension`, etc...

Reply:
Good idea, I'll work on something like this, maybe in the data source though so it's more easily testable 👍